src/RegaPoker.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: AGPL-3.0
+// The RegaPoker
+//
+// Copyright (C) 2020 Maker Ecosystem Growth Holdings, INC.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.6.12;
+
+interface IlkReg {
+    function list() external returns (bytes32[] memory);
+    function pip(bytes32) external returns (address);
+}
+
+interface Chainlog {
+    function getAddress(bytes32) external returns (address);
+}
+
+contract RegaPoker {
+
+    string   constant osig = "poke(bytes32)";
+    string   constant ssig = "poke()";
```

Comment:
I think these are backwards, looking at how they are used? (see lines 47 and 51)